src/cli.ts

```diff
@@ -31,33 +30,30 @@ function parseArguments(): Arguments {
     .usage('Analyze and debug space usage through source maps.')
     .usage('Usage:')
     .usage(
-      '$0 script.js [script.js.map] [--json | --html | --tsv | --file map.html] [-m | --only-mapped] [--replace=BEFORE_1 BEFORE_2 --with=AFTER_1 AFTER_2] [--no-root] [--version] [--help | -h]'
+      '$0 script.js [script.js.map] [--json [result.json] | --html [result.html] | --tsv [result.csv]] [-m | --only-mapped] [--replace=BEFORE_1 BEFORE_2 --with=AFTER_1 AFTER_2] [--no-root] [--version] [--help | -h]'
     )
     .example('$0 script.js script.js.map', 'Explore bundle')
     .example('$0 script.js', 'Explore bundle with inline source map')
     .example('$0 dist/js/*.*', 'Explore all bundles inside dist/js folder')
     .demandCommand(1, 'At least one js file must be specified')
     .options({
       json: {
-        type: 'boolean',
-        description: 'Output JSON (on stdout) instead of generating HTML and opening the browser.',
-        conflicts: ['tsv', 'html', 'file'],
+        type: 'string',
```

Comment:
What indicates to yargs that the string (file name) is optional? If I specify `--html --only-mapped` does it interpret the `--only-mapped` as a file name or another argument?

Reply:
I guess yargs smart enough to separate option (`--only-mapped`) from the value (empty string). I should check it